### ISSUE TITLE
Can I use another scheduler ?

### ISSUE THREAD

**Issue by pengcheng4451** [WeBankFinTech/DataSphereStudio#77]
Can I replace azkaban with another scheduler? Like  airflow or easysheduler.

**Comment by wushengyeyouya** [comment]
Yes, just implement a new SchedulerAppJoint, then all of DSS workflow nodes can be scheduled in new scheduler.

How does the DSS workflow publish to Scheduler? Please see:  https://github.com/WeBankFinTech/DataSphereStudio/blob/master/docs/zh_CN/ch4/DSS%E5%B7%A5%E7%A8%8B%E5%8F%91%E5%B8%83%E8%B0%83%E5%BA%A6%E7%B3%BB%E7%BB%9F%E6%9E%B6%E6%9E%84%E8%AE%BE%E8%AE%A1.md

How to integrate a new scheduler? Please see: https://github.com/WeBankFinTech/DataSphereStudio/blob/master/docs/zh_CN/ch4/%E7%AC%AC%E4%B8%89%E6%96%B9%E7%B3%BB%E7%BB%9F%E6%8E%A5%E5%85%A5DSS%E6%8C%87%E5%8D%97.md

**Comment by YinDonglai** [comment]
提供最新版DataSphereStudio全家桶安装服务。
包括（DataSphereStudio, Linkis, Exchangeis(Datax, Sqoop), Scriptis, Streamis(Flink), Visualis, DolphinScheduler)
1. 另外包含 元数据，数据安全，数据质量，数据血缘。（这些都是DataSphereStudio不自带的服务）
2. 包含hadoop集群搭建（hive, spark, flink, kafak, kylin）
3. 提供上述所有系统二次定制开发服务
4. 提供系统搭建运维疑难问题解决
![本人微信](https://site-css-test.oss-cn-hangzhou.aliyuncs.com/681723451208_.pic.jpg?x-oss-process=image/resize,w_500,h_300 "扫描二维码添加微信")
持续为DataSphereStudio开源贡献力量
